src/app/services/ws.service.ts

```diff
@@ -62,16 +60,6 @@ export class WebSocketService {
     return this.authStatus$.asObservable();
   }
 
-  get consoleMessages(): Observable<string> {
```

Comment:
This getter was causing the original issue in the ticket. It was basically subscribing to the same end point without changing the request id. Which caused an error and caused the ui to crash.